config/locales/javascript/javascript.en.yml

```diff
@@ -70,9 +70,17 @@ en:
           other: "<%= count %>ms"
         unknown: "unknown"
         not_available: "not available"
+        total:
+          one: "There is only one known pod."
+          other: "There are <%= count %> known pods in total."
         unchecked:
           one: "There is still one pod that hasn't been checked at all."
           other: "There are still <%= count %> pods that haven't been checked at all."
+        active:
+          one: "One pod is active."
+          other: "<%= count %> pods were active recently."
+        none_active: "None of them are active."
+        all_active: "All of them are active"
```

Comment:
This is missing the `.` at the end of the sentence.

Reply:
Oh, I also just noticed that these are in different times, so the `other` uses past with "were" while the others use "are" in present. I'd probably use "... were active recently" in all of the variants.